components/user-modal/component.js

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import Modal from 'components/modal';
+
+import { createUserEntry, updateUserEntry } from 'utils/airtable';
+
+import Step1 from './content/step1';
+import Step2 from './content/step2';
+
+import './style.scss';
+
+const UserModal = ({ open, onClose }) => {
+  const [step, setStep] = useState('step1');
+  const [userId, setUserId] = useState(null);
+  const [userData, setUserData] = useState({
+    job_role: '',
+    job_role_description: '',
+    map_usage: '',
+    map_usage_description: '',
+    email: '',
+  });
+
+  const handleCreateUser = async () => {
+    const user = await createUserEntry({ ...userData });
```

Comment:
`createUserEntry` can reject. Put a try/catch around it.

It'd be great if you could display an error message in the UI if this fails.